### PR TITLE
Added Email to project

### PR DESCRIPTION
**Pull request by isfandiyoro5**

### CODE REVIEW

File: TestHotel/EmailService/EmailSender.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmailService
+{
+    internal class EmailSender
```
Comment (mrIminjonov) [inline TestHotel/EmailService/EmailSender.cs]: ochirvor

File: TestHotel/EmailService/IEmailSender.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmailService
+{
+    internal interface IEmailSender
+    {
```
Comment (mrIminjonov) [inline TestHotel/EmailService/IEmailSender.cs]: buniyam

File: TestHotel/EmailService/Message.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmailService
+{
+    internal class Message
+    {
```
Comment (mrIminjonov) [inline TestHotel/EmailService/Message.cs]: buniyam

File: TestHotel/EmailService/EmailConfiguration.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmailService
+{
+    internal class EmailConfiguration
+    {
+    }
```
Comment (mrIminjonov) [inline TestHotel/EmailService/EmailConfiguration.cs]: ochirvor

**Review by mrIminjonov — CHANGES_REQUESTED** [review]

**Review by MirazamTUT — APPROVED** [review]

**Pull request merged**